spec/index.bs

```diff
@@ -9149,37 +9220,42 @@ command encoder can no longer be used.
         Completes recording of the commands sequence and returns a corresponding {{GPUCommandBuffer}}.
 
         <div algorithm=GPUCommandEncoder.finish>
-            **Called on:** {{GPUCommandEncoder}} |this|.
+            <div data-timeline=content>
+                **Called on:** {{GPUCommandEncoder}} |this|.
 
-            **Arguments:**
+                **Arguments:**
 
-            <pre class=argumentdef for="GPUCommandEncoder/finish(descriptor)">
-                descriptor:
-            </pre>
+                <pre class=argumentdef for="GPUCommandEncoder/finish(descriptor)">
+                    descriptor: Reserved for future use.
```

Comment:
The descriptor actually does one thing right now, which is set the label. Otherwise we would just not have it.

Which reminds me we'll have to think of a not-too-messy way of setting the label on every object we create.